<a href="https://colab.research.google.com/github/Gauravmehra1/play-store-app-analysis/blob/main/Play_Store_App_Review_Analysis_Capstone_Project.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

## <b> The Play Store apps data has enormous potential to drive app-making businesses to success. Actionable insights can be drawn for developers to work on and capture the Android market. </b>

## <b> Each app (row) has values for catergory, rating, size, and more. Another dataset contains customer reviews of the android apps.</b>

## <b> Explore and analyze the data to discover key factors responsible for app engagement and success. </b>

**Import Libraries**

In [13]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns

**Importing data from google colab** 

In [14]:
from google.colab import drive
drive.mount('/content/drive')

Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount("/content/drive", force_remount=True).


In [15]:
File ='/content/drive/MyDrive/EDA/PlayStore.csv'
df=pd.read_csv(File)

**Checking columns , shape and first 5 element** 

In [16]:
df.columns


Index(['App', 'Category', 'Rating', 'Reviews', 'Size', 'Installs', 'Type',
       'Price', 'Content Rating', 'Genres', 'Last Updated', 'Current Ver',
       'Android Ver'],
      dtype='object')

In [17]:
df.shape

(10841, 13)

In [ ]:
df.head()

**Play store data frame has 13 rows and 10841 columns**

**App**- It will tell about the name of apps in data frame 

**category** -It tells the app catagory 

**Rating** - In this we will know about the rating of app 

**Reviews** - It tell us the reviews give by the users 

**Size** - It tells us about the size of application in our moblie phone 

**Installs** - It will lets us know about the downloads

**Type**- Tell us the app is free or paid 

**Price**- It will  give us the app price and for free app it will give Zero

**content rating** - It will tell us the app can be used by everyone or by teen 

**Genres**-it tells us about the various other categories to which an application can belong.

**Last updated**- It tells us when app updated 

**Current ver**- It tells us the updated version of the app 

**Android ver**- It tell us which android version is used by the app












In [ ]:
#taking info of our data frame 
df.info()

In [ ]:
#Finding the duplicate record in data frame 
df.duplicated().sum()

In [ ]:
# printing duplicate rows and coloumn 
df[df.duplicated()]

**we will take data of all Duplicate app and will drop them**

In [ ]:
df.drop_duplicates()

In [ ]:
df.drop_duplicates(subset=['App'])
